todo_app/app.py

```diff
@@ -1,15 +1,35 @@
 from flask import Flask
-
+from flask import request
 from todo_app.flask_config import Config
+from flask import render_template
+from todo_app.data import session_items
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+@app.route('/todo')
```

Comment:
The text in the top left of the header "To-Do App" actually links to "/" (`href="/"`)

Also, the README.md still says to go to localhost:5000